docs/decoder.md

```diff
@@ -68,27 +73,35 @@ See [Edge Server PubSub Message Schema](edge.md#edge-server-pubsub-message-schem
 
 ### Decoded message metadata schema
 
-Schema of the metadata object that is added to the message:
+Decoded messages published to Pub/Sub will contain the following attributes:
 
 ```
-required group metadata {
-  required string document_version           // from uri for non-Telemetry, from message for Telemetry
-  required string document_id                // from uri
-  required string geo_country                // from geoip lookup
-  required string geo_subdivision1           // from geoip lookup
-  required string geo_subdivision2           // from geoip lookup
-  required string geo_city                   // from geoip lookup
-  required string submission_timestamp       // from edge metadata
-  optional string date                       // header from client
-  optional string dnt                        // header from client
-  optional string x_pingsender_version       // header from client
-  optional string user_agent_browser         // from user_agent
-  optional string user_agent_browser_version // from user_agent
-  optional string user_agent_os              // from user_agent
-  optional string user_agent_os_version      // from user_agent
-}
+document_version           // from uri for non-Telemetry, from message for Telemetry
```

Comment:
I note that the fields are no longer marked as optional or required and types are also omitted. For consistency with the earlier error message specification it may be good to have the two match.

Reply:
will do